sp/src/vscript/vscript_squirrel.cpp

```diff
@@ -827,7 +829,7 @@ void PushVariant(HSQUIRRELVM vm, const ScriptVariant_t& value)
 		sq_createinstance(vm, -1);
 		SQUserPointer p;
 		sq_getinstanceup(vm, -1, &p, 0);
-		new(p) Vector(value);
+		new(p) Vector(static_cast<const Vector&>(value));
```

Comment:
I find this one interesting.
`Vector()` has an overload for `vec_t` (i.e. `float`) and `ScriptVariant_t` can be cast to both `Vector&` and `float`. gcc regards this as an unresolvable ambiguity. I guess MSVC doesn't, and I'm guessing a cast to `Vector&` is what's desired here.

Reply:
Ya, this cast is desirable, unfortunately MSVC doesn't conform with the standard for these conversions and allows implicit conversions here. (Compiling with conforming options on MSVC will normally pick these up)